netci.groovy

```diff
@@ -1624,6 +1656,7 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
             switch (architecture) {
                 case 'x64':
                 case 'x86':
+                case 'x86lb':
                     if (architecture == 'x86' && os == 'Ubuntu') {
```

Comment:
not sure it that is needed.

It looks like the line this comment was attached to has disappeared or moved, so maybe you changed the code, or maybe it is no longer relevant.